### DISCUSSION TITLE
Spam which appears to indicate my email address has been harvested fro Github

### DISCUSSION THREAD

**Discussion by JillThornhill** [community/community#132205] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi. I have received a spam email which I suspect is an attempt at fraud. Since it references my Github account, it looks as if my email address has been harvested from GitHub. I can supply further details if needed. Firstly, I would like to warn other Github users that hackers may have accessed private GitHub profile information. Secondly, I would like to find out how to alert GitHub to a possible security vulnerability.

**Comment by ebndev** [comment]
Hi @JillThornhill, 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
